### PR TITLE
(MAINT) - Remove github_changelog gem footprint

### PR DESCRIPTION
**Pull request by Ramesh7**
## Summary

Cleanup activity - Removing the reference of github changelog.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [x] Manually verified.

### CODE REVIEW

File: moduleroot/.rubocop.yml.erb [hunk 1]
```diff
@@ -6,8 +6,9 @@ defaults = {
     'rubocop-rspec',
   ],
   'AllCops' => {
+    'NewCops' => 'enable',
     'DisplayCopNames' => true,
-    'TargetRubyVersion' => '2.6',
+    'TargetRubyVersion' => '2.7',
```
Comment (bastelfreak) [inline moduleroot/.rubocop.yml.erb:11]: Why is that changed? It doesn't relate to the PR title

Comment (Ramesh7) [reply moduleroot/.rubocop.yml.erb:11]: we manually change the ruby version on PDK update everytime, so thought to cover as part of this as its small and quick one.
Let me know if I need to cover as part of diff PR?

Comment (jordanbreen28) [reply moduleroot/.rubocop.yml.erb:11]: I think this needs to stay as ruby 2.6, to keep compatibility with the jruby versions still used in puppet server (IIRC)

Comment (Ramesh7) [reply moduleroot/.rubocop.yml.erb:11]: ahh i see, wasn't knowing that. Would like to know more about it. Thanks for pitching in @jordanbreen28.

Comment (Ramesh7) [reply moduleroot/.rubocop.yml.erb]: resolved, ready for review again.

File: moduleroot/.rubocop.yml.erb [hunk 2]
```diff
@@ -6,6 +6,7 @@ defaults = {
     'rubocop-rspec',
   ],
   'AllCops' => {
+    'NewCops' => 'enable',
```
Comment (bastelfreak) [inline moduleroot/.rubocop.yml.erb:9]: I still think the rubocop config should not be changed here. The change isn't related to removing the changelog generator. Also I don't know if the current rubocop config handles all cops, enabling new cops could be a breaking change (that also depends on how you manage newer rubocop versions).

Comment (Ramesh7) [reply moduleroot/.rubocop.yml.erb]: done!!

**Review by LukasAud — APPROVED** [review]
LGTM
